src/main/java/io/quarkus/gizmo2/creator/ops/StringBuilderOps.java

```diff
@@ -67,8 +67,7 @@ public StringBuilderOps append(final Expr expr) {
             case "Ljava/lang/String;" -> invokeInstance(StringBuilder.class, "append", String.class, expr);
             case "Ljava/lang/CharSequence;" -> invokeInstance(StringBuilder.class, "append", CharSequence.class, expr);
             default -> invokeInstance(StringBuilder.class, "append", Object.class, expr);
-        }
-        return this;
+        });
```

Comment:
Any reason for no longer returning `this`? It probably doesn't make a difference, but the method is literally documented to `@return this instance`.

Reply:
Yes, it fails if the wrapper object was created from an arbitrary expression since `this` would represent an object which is no longer on the stack.

The wrapper objects are set up to only be used once. The `StringBuilderOps` is a sort of weird exception in that there is special code to create a variable for it, but it's not guaranteed that its receiver will be in a variable.